CustomScript/customscript.py

```diff
@@ -103,37 +103,81 @@ def enable(hutil):
     Ensure the same configuration is executed only once
     If the previous enable failed, we do not have retry logic here,
     since the custom script may not work in an intermediate state.
-    But if download_files fails, we will retry it for maxRetry times.
+    But if download_files fails, we will wait and retry.
     """
     hutil.exit_if_enabled()
     prepare_download_dir(hutil.get_seq_no())
-    maxRetry = 2
-    for retry in range(0, maxRetry + 1):
+    public_settings = hutil.get_public_settings()
+    if public_settings is None:
+        raise ValueError("Public configuration couldn't be None.")
+    tries = public_settings.get('tries')
```

Comment:
Here I followed the options of the command "wget".
```
Download:
  -t,  --tries=NUMBER            set number of retries to NUMBER (0 unlimits).
  -w,  --wait=SECONDS            wait SECONDS between retrievals.
```
If we used `retrycount`, then `retrycount` should be `tries -1`.